### PR TITLE
cb-208 create `<UnfulfilledRequirements />` component

### PR DESCRIPTION
**Pull request by Andresbl123**
Implementar el componente UnfulfilledRequirements

image

### CODE REVIEW

File: src/components/cards/UnfulfilledRequirements/UnfulfilledRequirements.stories.tsx
Comment (novadeveloper14) [inline src/components/cards/UnfulfilledRequirements/UnfulfilledRequirements.stories.tsx:1]: the story should be a folder called stories

File: src/components/cards/UnfulfilledRequirements/UnfulfilledRequirements.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import { Meta, StoryObj } from "@storybook/react";
+import { UnfulfilledRequirements } from ".";
+
+type Story = StoryObj<typeof UnfulfilledRequirements>;
+
+const meta: Meta<typeof UnfulfilledRequirements> = {
+  component: UnfulfilledRequirements,
+  title: "components/alerts/UnfulfilledRequirements",
+  argTypes: {
```
Comment (novadeveloper14) [inline src/components/cards/UnfulfilledRequirements/UnfulfilledRequirements.stories.tsx:9]: This information must be obtained from a file called props.ts

File: src/components/cards/UnfulfilledRequirements/index.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import { MdWarningAmber } from "react-icons/md";
+import { Divider } from "@inubekit/divider";
+import { Icon } from "@inubekit/icon";
+import { Stack } from "@inubekit/stack";
+import { Text } from "@inubekit/text";
+
+import { Fieldset } from "@src/components/data/Fieldset";
```
Comment (novadeveloper14) [inline src/components/cards/UnfulfilledRequirements/index.tsx:7]: use the `@components` alias directly

File: src/components/data/Fieldset/Fieldset.stories.tsx
Comment (novadeveloper14) [inline src/components/data/Fieldset/Fieldset.stories.tsx:1]: the story should be a folder called stories

File: src/components/data/Fieldset/Fieldset.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+import { Text } from "@inubekit/text";
+
+import { Fieldset } from ".";
+
+type Story = StoryObj<typeof Fieldset>;
+
+const meta: Meta<typeof Fieldset> = {
+  component: Fieldset,
+  title: "components/data/Fieldset",
+  argTypes: {
```
Comment (novadeveloper14) [inline src/components/data/Fieldset/Fieldset.stories.tsx:13]: This information must be obtained from a file called props.ts in the stories folder

File: src/components/data/Fieldset/index.tsx [hunk 4]
```diff
@@ -0,0 +1,79 @@
+import { MdAdd } from "react-icons/md";
+import { Text } from "@inubekit/text";
+import { Stack } from "@inubekit/stack";
+import { Button } from "@inubekit/button";
+import { StyledContainerFieldset } from "./styles";
```
Comment (novadeveloper14) [inline src/components/data/Fieldset/index.tsx:5]: organize imports

**Review by novadeveloper14 — CHANGES_REQUESTED** [review]

File: src/components/cards/UnfulfilledRequirements/index.tsx (hunk 2 above)
Comment (Andresbl123) [reply src/components/cards/UnfulfilledRequirements/index.tsx:7]: resuelto

File: src/components/data/Fieldset/index.tsx (hunk 4 above)
Comment (Andresbl123) [reply src/components/data/Fieldset/index.tsx:5]: resuelto

**Review by novadeveloper14 — APPROVED** [review]

**Review by carin91 — APPROVED** [review]

**Pull request merged**